app/serializers/user_notification_schedule_serializer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class UserNotificationScheduleSerializer < ApplicationSerializer
+  attributes :id, :user_id, :enabled
+
+  7.times do |n|
+    attribute :"day_#{n}_start_time"
+    attribute :"day_#{n}_end_time"
+  end
```

Comment:
```suggestion
  attribute :"day_1_start_time"
  attribute :"day_1_end_time"
  attribute :"day_2_start_time"
  attribute :"day_2_end_time"
  attribute :"day_3_start_time"
  attribute :"day_3_end_time"
  attribute :"day_4_start_time"
  attribute :"day_4_end_time"
  attribute :"day_5_start_time"
  attribute :"day_5_end_time"
  attribute :"day_6_start_time"
  attribute :"day_6_end_time"
  attribute :"day_7_start_time"
  attribute :"day_7_end_time"
```

I think we can just be abit more verbose here for readability.

Reply:
Sure, they start with 0 tho. I'll push the change.